src/playlist-loader.js

```diff
@@ -363,8 +363,6 @@ export default class PlaylistLoader extends EventTarget {
       playlist = this.master.playlists[playlist];
     }
 
-    window.clearTimeout(this.mediaUpdateTimeout);
```

Comment:
This causes livestreams to timeout after the first couple of seconds and timeout. I think we should not clear this timeout here and just let `media()` run at the timeout unless it is cleared somewhere else. See #396